daemon/cmd/fqdn.go

```diff
@@ -522,7 +522,7 @@ func (d *Daemon) notifyOnDNSMsg(lookupTime time.Time, ep *endpoint.Endpoint, epI
 			AnswerTypes:       recordTypes,
 		}),
 	)
-	record.Log()
+	defer record.Log()
```

Comment:
nit: Could also move the record formation and logging to the end to this function, as there are no `return`s in between?